### ISSUE TITLE
The specaug applied in whisper encoder seems wrong

### ISSUE THREAD

**Issue by zhenhaoyong** [espnet/espnet#5198]
The shape of feats after `log_mel_spectrogram` is `[B, 80, T]`. However, the `specaug` requires the shape of input to be `[B, T, F]`.

        feats, feats_lens = self.log_mel_spectrogram(xs_pad, ilens)

        if self.specaug is not None and self.encoders.training:
            feats, feats_lens = self.specaug(feats, feats_lens)

        xs_pad, olens = self.whisper_encode(feats, feats_lens)

```
def time_warp(x: torch.Tensor, window: int = 80, mode: str = DEFAULT_TIME_WARP_MODE):
    """Time warping using torch.interpolate.

    Args:
        x: (Batch, Time, Freq)
        window: time warp parameter
        mode: Interpolate mode
    """
```

**Comment by sw005320** [comment]
Thanks for the report.
@siddhu001, can you confirm it?

**Comment by siddhu001** [comment]
Yes this is a known issue. I made a small PR https://github.com/espnet/espnet/pull/5206 to quickly fix this.
